Products/CMFPlone/resources/browser/mixins.py

```diff
@@ -67,9 +68,10 @@ def __call__(self):
             less_vars_params[name] = value
 
         for name, value in registry.items():
-            t = value.format(**less_vars_params)
+            t = SafeFormatter(value).safe_format(**less_vars_params)
```

Comment:
@mauritsvanrees you have imported ``safe_format`` but using ``SafeFormatter``. This leads to an error mentioned here: https://github.com/plone/Products.CMFPlone/pull/1938#issuecomment-277922063